api/src/main/java/com/exelaration/abstractmemery/controllers/MemeController.java

```diff
@@ -0,0 +1,29 @@
+package com.exelaration.abstractmemery.controllers;
+
+import com.exelaration.abstractmemery.domains.Caption;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.exelaration.abstractmemery.domains.Caption;
+
+@RestController
+@RequestMapping("/meme")
+@CrossOrigin(origins = "http://localhost:3000")
+public class MemeController {
+
+    @PostMapping("/")
+    public ResponseEntity newCaption(@RequestBody Caption caption) {
```

Comment:
Since we renamed the controller to be more focused on the fact that it's all about a meme, we should rename this function as well.  I think `addMeme` is appropriate.